include/ck/tensor_operation/gpu/device/impl/device_grouped_gemm_xdl_splitk_cshuffle.hpp

```diff
@@ -311,7 +309,7 @@ struct DeviceGroupedGemmXdlSplitKCShuffle : public DeviceGroupedGemmSplitK<ALayo
                                            k0_padded,
                                            K_BATCH};
 
-                gemm_kernel_args_.emplace_back(
+                gemm_kernel_host_args_[i] = GemmTransKernelArg(
```

Comment:
This line would be problematic. Cause you call `Argument` constructor before setting host pinned allocated memory.